### ISSUE TITLE
ObjectExplorer not work on x64

### ISSUE THREAD

**Issue by computerline1z** [REhints/HexRaysCodeXplorer#10]
As I use function Object Explorer in x64 PE I got 2 issue

First : The IDA crash when I click on the context menu, as I see the debug log, I figureout it error in the format string at line 271 - tmp.cat_sprnt(" 0x%x:  %s", rtd, name); in ObjectExplorer.cpp, I reformat the output to - tmp.cat_sprnt(_T(" 0x%I64X:  %s"), rtd, name); and recompile then the plugin work ok.

Second : I see the vtable reference in PE file compile with vs2013 x64 use LEA instruction insteal of  MOV intruction as in x32
```
lea     rcx, off_14000E2A8
```
so the code at line 77 in ObjectExplorer.cpp will not work
```
BOOL get_vtbl_info(ea_t ea_address, VTBL_info_t &vtbl_info) {
.......................................

					if((*((PUINT) disasm_line) == 0x20766F6D /*"mov "*/) && (strstr(disasm_line+4, " offset ") != NULL))
					{
						is_move_xref = TRUE;
						break;
					}
......................................
} 
```
Thanks for aweasome plugin !

**Comment by matrosov** [comment]
Thank you for your bug submission!

**Comment by matrosov** [comment]
Fixed in [v1.7 [NSEC Edition]](https://github.com/REhints/HexRaysCodeXplorer/releases/tag/1.7)

**Comment by egirault** [comment]
Hi,

It seems that the vtable identification feature still does not work on x64 binaries; type reconstruction works but vtables are not identified and integrated to the reconstructed type. 

Indeed, the `get_vtbl_info` method uses 32-bit memory access primitives, such as `isDwrd` (line 100 in ObjectExplorer.cpp) and `get_32bit` (line 103). There are also 32-bit sizes used in that function (`sizeof(UINT)`) and in `create_vtbl_struct` (the `4` constant is hardcoded at lines 148, 228, 239). 

I guess the best would be to add size parameter to those functions in order to handle both 32-bit and 64-bit.

**Comment by computerline1z** [comment]
@egirault You could try my fork, I modified some to work with ida6.6 and 64bit https://github.com/computerline1z/HexRaysCodeXplorer

Wait for next HexRaysCodeXplorer release :)

**Comment by matrosov** [comment]
fixed in v2.0 [BlackHat Edition] ;)
